core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableInfo.java

```diff
@@ -316,4 +316,18 @@ public static TableInfo deserialize(byte[] bytes) throws IOException {
     return parentRelationIdentifiers;
   }
 
+  /**
+   * whether this table has aggregation DataMap or not
+   */
+  public boolean hasAggregationDataMap() {
```

Comment:
I think it is ok you move this logic to CarbonTable.hasAggregationDataMap directly. We better keep TableInfo unmodified

Reply:
ok